generate.sh

```diff
@@ -0,0 +1,12 @@
+git clone https://github.com/Snektron/vulkan-zig --depth 1
+zig run vulkan-zig/generator/main.zig -- vk.xml vk.zig
+
+echo "
+// -------------------------------------------
+
+pub const ANativeWindow = opaque{};
+pub const CAMetalLayer = opaque{};
+pub const AHardwareBuffer = opaque{};
+" >> vk.zig
+
+zig fmt vk.zig
```

Comment:
@alichraghi sorry I missed this PR, there are conflicts now I think. But what is this? Why are we manually patching vk.zig? that's pretty weird, I don't feel good about it

Reply:
there's a weird definition of `ANativeWindow` and `AHardwareBuffer` in v1.x, which vulkan-zig can't understand so we get like `7 use of undeclared identifier` error.